web/public/locales/en.json

```diff
@@ -634,6 +634,14 @@
     "FO": {
       "zoneName": "Faroe Islands"
     },
+    "FO-NI": {
+      "countryName": "Faroe Islands",
+      "zoneName": "Northern Islands"
```

Comment:
I'm not totally sold on this name but I haven't found a better alternative yet.

Reply:
For the south it's easy: https://en.wikipedia.org/wiki/Su%C3%B0uroy -> Should translate to South Island
For the north it's a bit more tricky: meginøkið seems to mean something closer to "capital" (https://hagstova.fo/en/node/509 + google translate), so maybe it means like capital territory. But I guess Northern islands is fairer as https://en.wikipedia.org/wiki/Regions_of_the_Faroe_Islands shows that it groups all regons that are not the south island

Reply:
Isn't Northern Island just these 6 though?
<img width="283" alt="image" src="https://user-images.githubusercontent.com/30777521/184648346-d7b30074-2764-46b6-8434-a35fabf45781.png">
That is what I got from my research at least so it would be better to have a more distinctive name.
But I wouldn't say Northern Islands are completely wrong either.

Reply:
Ok I've asked an icelandic friend of mine. Even though icelandic != feroese, it's pretty close. His take was that it means "main islands"

Reply:
Okay then I think we should switch to Main Islands. I'll update the translations and zone keys to match later today.